content/en/docs/concepts/services-networking/service.md

```diff
@@ -619,6 +619,82 @@ You can integrate with [Gateway](https://gateway-api.sigs.k8s.io/) rather than S
 can define your own (provider specific) annotations on the Service that specify the equivalent detail.
 {{< /note >}}
 
+#### Node liveness impact on load balancer traffic
+
+Load balancer health checks are critical to modern applications. They are used to
+determine which VM instance the load balancer should dispatch traffic on. The
+Kubernetes APIs do not define how health checks have to be implemented for Kubernetes
+managed load balancers, instead it's the cloud providers who implement the behavior.
+Load balancer health checks are extensively used within the context of supporting the
+feature `externalTrafficPolicy`. If `cluster` is specified all nodes are eligible
+load balancing targets _as long as_ the node is not being deleted and kube-proxy
+is healthy. In this mode: load balancer health checks are configured to
+target the service proxy's readiness port and path. In the case of kube-proxy this
```

Comment:
IMO: the detail about kube-proxy belongs in https://kubernetes.io/docs/reference/networking/virtual-ips/

I'd move all the new text from here onwards into that other page.